ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -672,11 +672,10 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = "AppIcon-dev";
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "Apple Development";
```

Comment:
I assume this is only for local dev and not the build versions of the app? :) As they need to be manual with fastlane

Reply:
Only for local dev indeed (debug-development)